debian/control

```diff
@@ -1,23 +1,24 @@
 Source: noderedrevpinodes-server
 Section: misc
 Priority: optional
-Maintainer: Philipp Rosenberger <p.rosenberger@kunbus.com>
-Build-Depends: debhelper (>= 12)
-Standards-Version: 4.6.0
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Rules-Requires-Root: no
 Homepage: https://github.com/erminas/noderedrevpinodes-server
 Vcs-Browser: https://github.com/RevolutionPi/noderedrevpinodes-server
 Vcs-Git: https://github.com/RevolutionPi/noderedrevpinodes-server.git
+Build-Depends: debhelper-compat (= 12), dh-python, python3
```

Comment:
```suggestion
Build-Depends: debhelper-compat (= 12), dh-python
```
python3 is a dependency of dh-python. If the upstream project uses setup tools, we have to add python3-all, but they don't even have a setup.py file :D